davegnukem.txt

```diff
@@ -0,0 +1,12 @@
+[Section]
+Name = Dave Gnukem
+Version = 1.01
+License = MIT GPL CC
+LicenseInfo = 1
+Description =  Dave Gnukem is a retro-style 2D scrolling platform shooter. It is inspired by and similar to Duke Nukem 1.
+Category = 4
+URLSite = https://github.com/davidjoffe/dave_gnukem
+URLDownload = https://sourceforge.net/projects/gnukem/files/gnukem/Dave_Gnukem_Setup_1.0.1.exe
```

Comment:
Did you test this url inside reactos?
I doubt that it will actually download the binary.

Reply:
Yes, I just took a screenshot of it working., check it out.

![Screenshot from 2020-09-17 08-29-08](https://user-images.githubusercontent.com/5752734/93470552-151f3f00-f8c0-11ea-862c-18e9151b775d.png)